File: app/src/main/java/yifimovies/tittojose/me/yifi/api/model/Torrent.java
package yifimovies.tittojose.me.yifi.api.model;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

/**
 * Created by titto.jose on 20-12-2017.
 */

public class Torrent implements Serializable {
    @SerializedName("url")
    private String url;

    @SerializedName("hash")
    private String hash;

    @SerializedName("quality")
    private String quality;

    @SerializedName("seeds")
    private Integer seeds;

    @SerializedName("peers")
    private Integer peers;

    @SerializedName("size")
    private String size;

    @SerializedName("date_uploaded")
    private String dateUploaded;

    @SerializedName("date_uploaded_unix")
    private Integer dateUploadedUnix;

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    public String getHash() {
        return hash;
    }

    public void setHash(String hash) {
        this.hash = hash;
    }

    public String getQuality() {
        return quality;
    }

    public void setQuality(String quality) {
        this.quality = quality;
    }

    public Integer getSeeds() {
        return seeds;
    }

    public void setSeeds(Integer seeds) {
        this.seeds = seeds;
    }

    public Integer getPeers() {
        return peers;
    }

    public void setPeers(Integer peers) {
        this.peers = peers;
    }

    public String getSize() {
        return size;
    }

    public void setSize(String size) {
        this.size = size;
    }


    public String getDateUploaded() {
        return dateUploaded;
    }

    public void setDateUploaded(String dateUploaded) {
        this.dateUploaded = dateUploaded;
    }

    public Integer getDateUploadedUnix() {
        return dateUploadedUnix;
    }

    public void setDateUploadedUnix(Integer dateUploadedUnix) {
        this.dateUploadedUnix = dateUploadedUnix;
    }
}

File: app/src/main/java/yifimovies/tittojose/me/yifi/api/model/Torrent.kt
package yifimovies.tittojose.me.yifi.api.model

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

/**
 * Created by titto.jose on 20-12-2017.
 */
@Parcelize
class Torrent(
        @SerializedName("url")
        var url: String? = null,

        @SerializedName("hash")
        var hash: String? = null,

        @SerializedName("quality")
        var quality: String? = null,

        @SerializedName("seeds")
        var seeds: Int? = null,

        @SerializedName("peers")
        var peers: Int? = null,

        @SerializedName("size")
        var size: String? = null,

        @SerializedName("date_uploaded")
        var dateUploaded: String? = null,

        @SerializedName("date_uploaded_unix")
        var dateUploadedUnix: Int? = null
) : Parcelable